### PR TITLE
Updated links in covpass

### PR DESCRIPTION
**Pull request by RaulMeer**

### CODE REVIEW

File: twine-cert.txt [hunk 1]
```diff
@@ -3459,8 +3463,8 @@
     en = Add a valid certificate or renew your latest certificate to get information about your vaccination status.
     
 [infschg_more_info_link]
-    de = #Mehr Informationen::https://www.digitaler-impfnachweis-app.de/faq/#die-aktuell-h-ufigsten-fragen#
-    en = #More information::https://www.digitaler-impfnachweis-app.de/en/faq/#current-most-frequently-asked-questions#
+    de = #Mehr in den häufigen Fragen::https://www.digitaler-impfnachweis-app.de/faq/#die-aktuell-h-ufigsten-fragen#
```
Comment (Ein-Tim) [inline twine-cert.txt:3466]: Missing "Informationen"

File: twine-cert.txt [hunk 2]
```diff
@@ -3574,8 +3578,8 @@
     en = You are exempt from the mask requirement in some situations in this state; however, a mask is still recommended.\n\n *Rules from %@ (start of validity)
 
 [infschg_detail_page_no_mask_mandatory_link]
-    de = #Mehr Informationen::https://www.digitaler-impfnachweis-app.de/faq/#die-aktuell-h-ufigsten-fragen#
-    en = #More information::https://www.digitaler-impfnachweis-app.de/en/faq/#current-most-frequently-asked-questions#
+    de = #Mehr in den häufigen Fragen::https://www.digitaler-impfnachweis-app.de/faq/#die-aktuell-h-ufigsten-fragen#
```
Comment (Ein-Tim) [inline twine-cert.txt:3581]: Missing "Informationen"

File: twine-cert.txt [hunk 3]
```diff
@@ -3606,8 +3610,8 @@
     en = You are not exempt from the mask requirement in this state.\n\n *Rules from %@ (start of validity)
 
 [infschg_detail_page_mask_mandatory_link]
-    de = #Mehr Informationen::https://www.digitaler-impfnachweis-app.de/faq/#die-aktuell-h-ufigsten-fragen#
-    en = #More information::https://www.digitaler-impfnachweis-app.de/en/faq/#current-most-frequently-asked-questions#
+    de = #Mehr in den häufigen Fragen::https://www.digitaler-impfnachweis-app.de/faq/#die-aktuell-h-ufigsten-fragen#
```
Comment (Ein-Tim) [inline twine-cert.txt:3613]: Missing "Informationen"

File: twine-cert.txt [hunk 4]
```diff
@@ -3634,8 +3638,8 @@
     en = For this federal state the RKI has not received federal state-specifc rules. Therefore, only the uniform federal rules on mandatory masks apply.
 
 [infschg_detail_page_mask_status_uncertain_link]
-    de = #Mehr Informationen::https://www.digitaler-impfnachweis-app.de/faq/#die-aktuell-h-ufigsten-fragen#
-    en = #More information::https://www.digitaler-impfnachweis-app.de/en/faq/#current-most-frequently-asked-questions#
+    de = #Mehr in den häufigen Fragen::https://www.digitaler-impfnachweis-app.de/faq/#die-aktuell-h-ufigsten-fragen#
```
Comment (Ein-Tim) [inline twine-cert.txt:3641]: Missing "Informationen"

**Review by Ein-Tim — CHANGES_REQUESTED** [review]

**Review by RaulMeer — COMMENTED** [review]
changed wording for faq links after feedback

**Review by Ein-Tim — APPROVED** [review]
Looks good now!

**Pull request merged**